### PR TITLE
[doc] correct link to mailmap manual

### PR DESCRIPTION
**Pull request by ryenus**
the format of mailmap is explained in git-mailmap(5), not git-shortlog(1).

### CODE REVIEW

**Comment by ryenus** [comment]
@newren possible to take a look?

**Review by newren — CHANGES_REQUESTED** [review]
Hi!

Sorry for the lengthy delay.  :-(  The changes look good, but it would be nice if the commit message had a slightly longer explanation.  If you could replace
```
the format of mailmap is explained in git-mailmap(5), not git-shortlog(1).
```
with something like
```
Ever since git.git commit 42957af027 ("mailmap doc: create a new
"gitmailmap(5)" man page", 2021-01-12), meaning all Git versions >=
2.31, the documentation for the mailmap feature in Git has moved to a
separate git-mailmap(5) page; update our links accordingly.
```
that'd be great.  Also, is "ryenus" your actual real name, or just a user name?  If the latter, can you fix your author name and Signed-off-by accordingly (as per git.git guidelines)?  Thanks!

**Comment by newren** [comment]
Sad to lose the fix, but without a proper signoff, I can't accept.  Closing.  :-(

**Comment by ryenus** [comment]
@newren if *a proper signoff* is blocking this from being merged, please feel free to take the patch with whatever needed to get the fix in. After all it's just a trivial one.

**Pull request closed without merging**
